xxhash.c

```diff
@@ -391,15 +396,16 @@ XXH_PUBLIC_API unsigned int XXH32 (const void* input, size_t len, unsigned int s
 #else
     XXH_endianess endian_detected = (XXH_endianess)XXH_CPU_LITTLE_ENDIAN;
 
-#  if !defined(XXH_USELESS_ALIGN_BRANCH)
-    if ((((size_t)input) & 3) == 0)   /* Input is 4-bytes aligned, leverage the speed benefit */
+    if (XXH_FORCE_ALIGN_CHECK)
```

Comment:
The test was initially an `#ifdef` macro.
Now it's a regular `if`, with the expectation that compiler should remove the useless code segment (which is pretty reasonable).

Any specific reason you preferred the latest version ?